packages/fast_pipe_ppx/fast_pipe_ppx.0.0.1/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+synopsis: "Fast pipe, pipe first as a syntax transform"
+description: """
+Pipe first as a syntax transform.
+Provides a ppx to transform expressions containing the |. (Ocaml) or -> (Reason)
+operator. Pipes the left side as first argument to the right side.
+
+Example Reason:
+--------------
+/* validateAge(getAge(parseData(person))) */
+person
+->parseData
+->getAge
+->validateAge;
+
+/* Some(preprocess(name)); */
+name->preprocess->Some;
+
+/* f(a, ~b, ~c) */
+a->f(~b, ~c)
+
+Example Ocaml:
+--------------
+(* validateAge (getAge (parseData person)) *)
+person
+|. parseData
+|. getAge
+|. validateAge
+
+(* Some(preprocess name) *)
+name |. preprocess |. Some;
+
+(*f a ~b ~c *)
+a |. f ~b ~c
+"""
+maintainer: "Iwan Karamazow <m.falka@icloud.com>"
+authors: "Iwan Karamazow m.falka@icloud.com"
+license: "MIT"
+homepage: "https://github.com/IwanKaramazow/FastPipe"
+bug-reports: "https://github.com/IwanKaramazow/FastPipe"
+dev-repo: "git://github.com/IwanKaramazow/FastPipe.git"
+depends: [ 
+  "ocaml" {>= "4.02" & < "4.08"}
+  "reason" {>= "3.3.7"}
+  "dune"
```

Comment:
this dependency should be tagged as a `{build}` dependency

Reply:
Fixed!